google/cloud/telcoautomation/README.md

```diff
@@ -1,8 +1,8 @@
 # Telco Automation API C++ Client Library
 
 This directory contains an idiomatic C++ client library for the
-[Telco Automation API][cloud-service-docs], a service to APIs to automate 5G
-deployment and management of cloud infrastructure and network functions.
+[Telco Automation API][cloud-service-docs]. APIs to automate 5G deployment and
+management of cloud infrastructure and network functions.
```

Comment:
`s/. APIs to/, to/` perhaps.  Otherwise that last sentence is but a fragment.

Reply:
PTAL